modules/bootkube/resources/bootkube.sh

```diff
@@ -57,8 +56,9 @@ i=0
 while true; do
     set +e
     # shellcheck disable=SC2154,SC2086
-    /usr/bin/docker run \
+    /usr/bin/podman run \
         --rm \
+        --network host \
```

Comment:
Can you explain why you need this?

Reply:
It wasn't able to access the outside network without. I thought I heard that this was expected behavior with podman (though, I don't know where I heard that).

Reply:
this should have worked....

Reply:
podman should work fine without requiring --network=host.  What version of podman were you testing with?

Reply:
@rhatdan `podman version 0.7.3`

Reply:
> podman version 0.7.3

If this is a known issue with an upstream fix, we may be able to drop this after #253 takes us up to RHCOS 4.0.5595-1 with:

```console
$ curl -s http://aos-ostree.rhev-ci-vms.eng.rdu2.redhat.com/rhcos/images/cloud/4.0.5595-1/pkglist.txt | grep podman
 podman-0.8.5-2.gitdc5a711.el7.x86_64
```

Thank you openshift/os#288 :).